src/components/sky-nav/sky-nav.twig

```diff
@@ -12,7 +12,7 @@
     {# https://heydonworks.com/article/aria-label-is-a-xenophobe/ #}
     <span class="u-hidden-visually">Home: Cloud Four</span>
     {# http://simplyaccessible.com/article/7-solutions-svgs/#acc-heading-4 #}
-    {% include '../../assets/icons/cloud-four.svg.twig' with {
+    {% include '../../assets/brand-elements/cloud-four.svg.twig' with {
```

Comment:
> Or potentially could this be renamed to simply assets/logo.svg?

Perhaps! 

I liked the idea of placing the logo in `assets/brand` based on my assumption that we would or could be placing other brand assets such as the illustrated trees in that directory. 

I have updated this to `../../assets/brand/logo.svg.twig` but pending further discussion could be persuaded otherwise.